.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,18 @@
+---
+name: User story
+about: Create a user story and add it to the tracker
+title: "[STORY]: "
+labels: tracker
+assignees: ''
+
+---
+
+<!-- As an MLOps engineer I would like... -->
+
+### Sub-tasks
+
+- [ ] #(issue number)
+
+### A/C
+
+- Nothing broke
```

Comment:
same as above, a set of commented-out acceptance criteria.

Reply:
@devguyio @adelton Remove this too? otherwise please give some suggestions for "commented-out acceptance criteria".

Reply:
I think yea we should remove it as well